content/documentation/pages/5-feature-guides/2-streams/10-function-composition.md

```diff
@@ -184,19 +184,21 @@ We can now deploy a new stream by using the `http-transform` application that in
 stream create helloComposed --definition "http-transformer --server.port=9001 | log"
 ```
 
-To define the order in which the functions are run, we have to use the `spring.cloud.stream.function.definition` property to define the function definition.
+[//]: # 'TODO clarify that this is optional. Move optional to the end'
```

Comment:
Yeh, good call on verifying that it is "in fact" optional. Once you do this we can merge this in. Thanks for updating.

Reply:
should be `spring.cloud.function.definition`